src/modules/home/components/header/user-navigation.menu.tsx

```diff
@@ -66,27 +95,32 @@ export function UserNavigationMenu<T extends MenuProps>({ anchorEl, handleMenuCl
         transformOrigin={userNavigationStyles.menuContainer.transformOrigin}
         elevation={userNavigationStyles.menuContainer.elevation}
       >
-        <MenuItemLink onClick={handleMenuClose} to="/signup">
-          Sign up
-        </MenuItemLink>
-        <MenuItemLink onClick={handleMenuClose} to="/login" sx={{ borderBottom: '1px solid #DDDDDD' }}>
-          Log in
-        </MenuItemLink>
-        <MenuItemLink to="/wishlist" onClick={handleMenuClose}>
-          Wishlist
-        </MenuItemLink>
-        <MenuItemLink onClick={handleMenuClose} to="/giftcards">
-          Gift cards
-        </MenuItemLink>
+        {!auth.loggedIn && (
+          <Box>
+            <MenuItemLink onClick={handleMenuClose} to="/signup">
+              Sign up
+            </MenuItemLink>
+            <MenuItemLink onClick={handleMenuClose} to="/login" sx={{ borderBottom: '1px solid #DDDDDD' }}>
+              Log in
+            </MenuItemLink>
+          </Box>
+        )}
         <MenuItemLink onClick={handleMenuClose} to="/host/homes">
           Airbnb your home
         </MenuItemLink>
-        <MenuItemLink onClick={handleMenuClose} to="/host/experiences">
-          Host experience
-        </MenuItemLink>
-        <MenuItemLink onClick={handleMenuClose} to="/help">
-          Help center
+        <MenuItemLink to="/wishlist" onClick={handleMenuClose}>
+          Wishlist
         </MenuItemLink>
+        {auth.loggedIn ? (
+          <MenuItemLink
+            to="/"
```

Comment:
Remove this if you handle redirecting inside handleLogout or if there is no need to redirect.

Reply:
fixed